db/data/20200115145845_backfill_geckoboard_data.rb

```diff
@@ -0,0 +1,13 @@
+class BackfillGeckoboardData < ActiveRecord::Migration[6.0]
+  def up
+    submitted_claim_ids = Claim.submitted.pluck(:id)
+    paid_claim_ids = Claim.where.not(payment: nil).pluck(:id)
```

Comment:
I don't think this scope is correct. This will return all claims that have a payment _even those that have not actually been scheduled for payment_, e.g. any that are part of the current payroll run will be included, even though we haven't received or processed the payment confirmation report. We only want to backfill the claims that actually have a `scheduled_payment_date` set, right?

Reply:
Good call - I've changed this